Tasks/AzureResourceManagerTemplateDeploymentV3/operations/DeploymentScopeBase.ts

```diff
@@ -68,6 +69,14 @@ export class DeploymentScopeBase {
         }
     }
 
+    private getAzurePortalDeploymentURL() {
+        let portalUrl = this.taskParameters.endpointPortalUrl ? this.taskParameters.endpointPortalUrl : "https://portal.azure.com";
+        portalUrl += "/#blade/HubsExtension/DeploymentDetailsBlade/overview/id/";
+
+        let subscriptionSpecificURL = "/subscriptions/" + this.taskParameters.subscriptionId + "/resourceGroups/" + this.taskParameters.resourceGroupName + "/providers/Microsoft.Resources/deployments/" + this.taskParameters.deploymentName;
+        return portalUrl + subscriptionSpecificURL.replace(/\//g, '%2F');
```

Comment:
As you meantioned earlier, will ctrl+click work? If no, please explicitly mention that user should copy paste the url and open in browser window.

Reply:
Done